### ISSUE TITLE
Delimiter parsing in Vertica functions

### ISSUE THREAD

**Issue by kseniiaguzeeva** [dbeaver/dbeaver#12408]
#### System information: 
- DBeaver version 21.0.5 EA

#### Connection specification:
- Database name and version Vertica

#### Describe the problem you're observing:
We cannot create a function with delimiter inside its body. Can we find a way to recognise such delimiters by default?
https://www.vertica.com/blog/creating-vertica-functions-in-third-party-tools/

**Comment by LonwoLonwo** [comment]
may be connected with https://github.com/dbeaver/dbeaver/issues/11656

**Comment by LonwoLonwo** [comment]
For tests:

```sql
CREATE OR REPLACE FUNCTION dbvrTest(a VARCHAR, b INT) RETURN INT AS BEGIN RETURN 9; END;

CREATE OR REPLACE FUNCTION dbvrTest2(a VARCHAR,
b INT) RETURN INT AS
BEGIN
	RETURN
	(CASE
		WHEN b <= 5 THEN 1
		WHEN b <= 10 THEN 100-30 *(8 / 4)
		WHEN b <= 100 THEN 8 * 6
		ELSE 0
	END);
END;

CREATE FUNCTION myzeroifnull(x INT) RETURN INT
   AS BEGIN 
     RETURN (CASE WHEN (x IS NOT NULL) THEN x ELSE 0 END); 
END;
```

**Comment by LonwoLonwo** [comment]
Fixed in DBeaver 21.1.2

**Comment by uslss** [comment]
verified
